flake.nix

```diff
@@ -61,6 +61,14 @@
                 project.hsPkgs.hspec-discover.components.exes."hspec-discover"
                 project.hsPkgs.markdown-unlit.components.exes."markdown-unlit"
               ];
+              #   FIXME (peter-mlabs): This is currently broken.
+              #   In the future you can do something like
+              #
+              #     find -name '*.hs' -not -path './dist-*/*' -exec hlint -j --refactor --refactor-options="--inplace" {} +
```

Comment:
well nice! We can make a pre-commit-hooks module that does that. Not sure if it's safe to do such refactors.